backend/logger/file_handler.py

```diff
@@ -0,0 +1,87 @@
+import os
+import uuid
+import datetime
+import atexit
+
+
+class FileHandler:
+    """
+    In charge of handling files for the logger. This class will open, close,
+    and name files for the logger. Responsible for storing logs in safe place.
+    In addition, it will pass file handles to different programs.
+    """
+
+    def __init__(self, file_name):
+        self.file_name = file_name
+        self.full_name = file_name
+        self.file_handle = None
+
+        atexit.register(self.close_file)
```

Comment:
This is giving me an error when I run the tests: 
![image](https://user-images.githubusercontent.com/29286974/109371708-05a81700-789e-11eb-91a5-ffcb4355012d.png)

Reply:
Hmm, didn't get this issue on windows nor ubuntu. You can probably ignore that but I'll give it a look.